samza-core/src/main/java/org/apache/samza/config/ClusterManagerConfig.java

```diff
@@ -199,4 +205,12 @@ public boolean getJmxEnabled() {
       return true;
     }
   }
+
+  public boolean getJmxEnabledOnContainer() {
```

Comment:
Why do these need to be separate configs?

Reply:
`cluster-manager.jobcoordinator.jmx.enabled` is the config we use which can be misleading since it has jobcoordinator in it. I thought about it and wanted to rename this config to `cluster-manager.jmx.enabled` but we also have a legacy yarn config for the same. 

To avoid confusion I created a new, I agree one config for JC and Container but is renaming an existing config acceptable?

Reply:
Is there a need to enable/disable them separately? If not, let's use the existing config.